packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -119,6 +120,15 @@ abstract class BindingBase {
       );
     }
 
+    registerServiceExtension(
```

Comment:
Can you place this within the if above or the assert below so it’s tree-shaken out in release mode?

Reply:
Done